trino/auth.py

```diff
@@ -273,7 +273,8 @@ class _OAuth2TokenBearer(AuthBase):
     _BEARER_PREFIX = re.compile(r"bearer", flags=re.IGNORECASE)
 
     def __init__(self, redirect_auth_url_handler: Callable[[str], None]):
-        self._redirect_auth_url = redirect_auth_url_handler
+        self._redirect_auth_url = None
```

Comment:
Why would you want to store it?

Reply:
You are right. We don't need to store it. I'm removing this part